src/js/covidMap.js

```diff
@@ -422,6 +482,46 @@ fluid.defaults("fluid.covidMap.map", {
         filterCountOnMobile: {
             type: "fluid.covidMap.filterCount",
             container: "{that}.dom.filterCountOnMobile"
+        },
+        locationButtonOnMobile: {
```

Comment:
Perhaps the definition of these buttons could be moved into the mobile view handler?
The fact that "excludeSource" is required is I think a bug in the "fluid.button" implementation which we haven't run into so far since the buttons that we have so far were idempotent.
The 2nd definition looks incorrect since as I understand it the "expandedPanels" is a hash to boolean rather than an array.
I think we're getting to the point at which we might require some at least basic test cases - or at the least, a manual testing script instructions so that someone can quickly run through the app and verify that it is behaving as intended.